app/main/views/errors.py

```diff
@@ -13,47 +17,39 @@
 
 logger = logging.getLogger(__name__)
 
+errors_blueprint = Blueprint('errors', __name__)
```

Comment:
Should the errors blueprint be setting the cache headers like the root and questionnaire blueprints?